### PR TITLE
Modify ReadPCLVis3D sample to use .pcd from SampleData

### PR DESCRIPTION
**Pull request by chrisasc**
Updated sample to use .pcd from SampleData instead of your own point cloud. This makes the sample independent of CaptureWritePCLVis3D.

### CODE REVIEW

**Comment by robertkt1** [comment]
Not sure if the sample description at the top of file makes sense anymore. "To get a Zivid point cloud in PCD file format, run CaptureWritePCLVis3D sample."

If you run that sample you will not get a pcd file that you can use in this sample. In readZDF we advise them to our main page to get instructions for SampleData. Maybe we should do something similar here?

**Comment by SatjaSivcev** [comment]
> Not sure if the sample description at the top of file makes sense anymore. "To get a Zivid point cloud in PCD file format, run CaptureWritePCLVis3D sample."
> 
> If you run that sample you will not get a pcd file that you can use in this sample. In readZDF we advise them to our main page to get instructions for SampleData. Maybe we should do something similar here?

How about: The PCD file for this sample can be found under the main instructions for Zivid samples. Alternatively, run CaptureWritePCLVis3D sample.

Why can't the output of CaptureWritePCLVis3D be used @robertkt1?

**Comment by robertkt1** [comment]
And also this cannot be merged until we update SampleData with a pcd file

**Comment by robertkt1** [comment]
> 
> 
> > Not sure if the sample description at the top of file makes sense anymore. "To get a Zivid point cloud in PCD file format, run CaptureWritePCLVis3D sample."
> > If you run that sample you will not get a pcd file that you can use in this sample. In readZDF we advise them to our main page to get instructions for SampleData. Maybe we should do something similar here?
> 
> How about: The PCD file for this sample can be found under the main instructions for Zivid samples. Alternatively, run CaptureWritePCLVis3D sample.
> 
> Why can't the output of CaptureWritePCLVis3D be used @robertkt1?

Because CaptureWrite writes to the build directory and not to SampleData

**Comment by chrisasc** [comment]
> And also this cannot be merged until we update SampleData with a pcd file

This is already updated

**Comment by chrisasc** [comment]
> Not sure if the sample description at the top of file makes sense anymore. "To get a Zivid point cloud in PCD file format, run CaptureWritePCLVis3D sample."
> 
> If you run that sample you will not get a pcd file that you can use in this sample. In readZDF we advise them to our main page to get instructions for SampleData. Maybe we should do something similar here?

I agree! I will change that to something similar as ReadIterateZdf sample.

**Comment by chrisasc** [comment]
Updated the comments on top of the sample and tested it on Windows and Ubuntu with the latest SampleData.
Ready for next round.

File: source/Applications/Basic/Visualization/ReadPCLVis3D/ReadPCLVis3D.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
 This example shows how to read point cloud from PCL file and visualize it.
-To get a Zivid point cloud in PCD file format, run CaptureWritePCLVis3D sample.
+The ZDF file for this sample can be found under the main instructions for Zivid samples.
```
Comment (robertkt1) [inline source/Applications/Basic/Visualization/ReadPCLVis3D/ReadPCLVis3D.cpp:3]: Maybe a bit too similar as ReadIterateZdf? 
We don't use a ZDF in this sample

Comment (robertkt1) [reply source/Applications/Basic/Visualization/ReadPCLVis3D/ReadPCLVis3D.cpp:3]: The rest looks good

**Pull request merged**